app/connectors/BindingTariffFilestoreConnector.scala

```diff
@@ -72,4 +72,8 @@ class BindingTariffFilestoreConnector @Inject()(appConfig: FrontendAppConfig, ws
     }
   }
 
+  def get(attachment: Attachment)(implicit headerCarrier: HeaderCarrier): Future[Option[FilestoreResponse]] = {
+    http.GET[Option[FilestoreResponse]](s"${appConfig.bindingTariffFileStoreUrl}/file/${attachment.id}")
```

Comment:
Just checking that we're not able to use HttpVerbs here because of some limitation with FileStore?

Reply:
I think we had the HttpVerbs discussion when we did pdfs originally.  Don't remember the details, but there was some reason that they couldn't be used